libraries/chain/resource_limits.cpp

```diff
@@ -356,7 +356,7 @@ account_resource_limit resource_limits_manager::get_account_cpu_limit_ex( const
 
    uint128_t window_size = config.account_cpu_usage_average_window;
 
-   uint128_t virtual_cpu_capacity_in_window = (uint128_t)state.virtual_cpu_limit * window_size;
+   uint128_t virtual_cpu_capacity_in_window = (uint128_t)(elastic ? state.virtual_cpu_limit : config.cpu_limit_parameters.max) * window_size;
```

Comment:
Sorry I didnt catch this on first review but the math here is now out of sync with the assertions in `::add_transaction_usage` ideally those would be the same.  I know controller is externally maintaining in variants about billing but internal to the system that is the method that enforces limits.  `::get_account_*_limit` are supposed to be for informational purposes and not enforcement

Reply:
In case it was not clear, these methods are correct, but `:add_transaction_usage` is now out of date.